pkg/loadbalancer/loadbalancer.go

```diff
@@ -582,16 +580,15 @@ func NewBackend(id BackendID, protocol L4Type, ip net.IP, portNumber uint16) *Ba
 	return &b
 }
 
-// NewBackendWithState creates the Backend struct instance from given params,
-// and sets the restore state for the Backend.
+// NewBackendWithState creates the Backend struct instance from given params.
 func NewBackendWithState(id BackendID, protocol L4Type, ip net.IP, portNumber uint16,
-	state BackendState, restored bool) *Backend {
+	state BackendState) *Backend {
 	lbport := NewL4Addr(protocol, portNumber)
 	b := Backend{
-		ID:                   id,
-		L3n4Addr:             L3n4Addr{IP: ip, L4Addr: *lbport},
-		State:                state,
-		RestoredFromDatapath: restored,
+		ID:       id,
+		L3n4Addr: L3n4Addr{IP: ip, L4Addr: *lbport},
+		State:    state,
+		Weight:   1,
```

Comment:
Another thought occurs to me: Should the default weight be higher? 1000? 32768? That way one can decrease the weight of a single backend relative to the pool.

Also, the default weight should be a constant (even if it's 1) throughout the code.

Reply:
100 sounds great, thanks.